bob-apps/bin/bobt.rs

```diff
@@ -0,0 +1,421 @@
+use clap::{App, Arg, ArgMatches};
+use http::{StatusCode, Uri};
+use lazy_static::lazy_static;
+use rand::distributions::Uniform;
+use rand::prelude::*;
+use reqwest::RequestBuilder;
+use std::collections::BTreeMap;
+use std::time::Duration;
+use stopwatch::Stopwatch;
+
+const START_ID_ARG_NAME: &str = "start-id";
+const END_ID_ARG_NAME: &str = "end-id";
+const MAX_SIZE_ARG_NAME: &str = "size";
+const COUNT_ARG_NAME: &str = "key";
+const API_ADDRESS_ARG_NAME: &str = "api-address";
+const USERNAME_ARG_NAME: &str = "username";
+const PASSWORD_ARG_NAME: &str = "password";
+
+#[tokio::main]
+async fn main() {
+    env_logger::init();
```

Comment:
Will it be enough for the logs to be written to the console without additional settings?